package/README.md

```diff
@@ -1,44 +1,72 @@
-# GOV.UK Frontend - All components
+# GOV.UK Frontend
 
-All of the components in GOV.UK Frontend, in a single package.
+GOV.UK Frontend contains the code you need to start building a user interface
+for government platforms and services.
 
-## Guidance
+See live examples of GOV.UK Frontend components, and guidance on when to use
+them in your service, in the [GOV.UK Design
+System](https://govuk-design-system-production.cloudapps.digital/).
 
-Find out when to use the Back link component in your service in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital).
+## Contact the team
 
-## Dependencies
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
+team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
 
-To consume all components you must be running npm version 5 or above.
+## Requirements
+
+To use GOV.UK Frontend with NPM you must:
+
+Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM. The minimum version of Node required is 8.6.0.
 
 ## Installation
 
-    npm install --save govuk-frontend
+To install GOV.UK Frontend, run:
 
-## Requirements
+```
+npm install --save govuk-frontend
+```
+
+## Importing styles
+
+You need to import the GOV.UK Frontend styles into the main Sass file in your
+project. You should place the below code before your own Sass rules (or Sass
+imports) if you want to override GOV.UK Frontend with your own styles.
+
+To import add the below to your Sass file:
+
+  ```CSS
+  @import "node_modules/govuk-frontend/all";
+  ```
+
+[More details](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#importing-styles)
+
+## Importing JavaScript
+
+Some of the JavaScript included in GOV.UK Frontend improves the usability and
+accessibility of the components. You should make sure that you are importing and
+initialising it in your application to ensure that all users can properly use it
+successfully.
 
-### Build tool configuration
+You can include Javascript for all components either by copying the `all.js` from `node_modules/govuk-frontend` into your application or referencing the file directly:
```

Comment:
These changes are good but it makes me uneasy that they've not been made consistently in the installation section...